static_inventories/donbot_azure.yml

```diff
@@ -0,0 +1,67 @@
+---
+all:
+  children:
+    openstack_api:
+      hosts:
+        localhost:
+          ansible_python_interpreter: /usr/bin/env python
+    ldap_server:
+      hosts:
+        # LsAai lDAP
+        ladap:
+          host_networks:
+            - name: donbot_external_network
+              security_group: "{{ stack_prefix }}_cluster"
+          # Ldap settings
+          create_ldap: true
+          ldap_domains:
+            stack:
+              uri: "ldaps://{{ stack_prefix }}"
+              base: "dc={{ stack_name }},dc=local"
+              schema: rfc2307bis
+              min_id: 1000
+              max_id: 21999
```

Comment:
This conflicts with IDs we currently use for local admin accounts, which are created on the machine. Range was probably copy-pasted from experimental 'local' stack LDAP on Hyperchicken. We will need something else here and preferably it does not conflict with other ranges we already use:
* SRAM: `70000000 - 72999999`
* IDVault: `50100000 - 55999999`

Reply:
I just reused the existing LDAP settings. So I am not sure if this will be even used.

The ranges will be actually defined on LS-AAI itself and then copied to the LDAP. So are these even needed then? Or should we perhaps make them broader? For example between 1000 and 99999999. Or ...

We should have a chat.

Reply:
Ranges are used for `sssd`. When a UID/GID is outside the range for a certain IdP, then the entry is ignored. E.g. when we have multiple IdPs like IDVault LDAP, SRAM LDAP and LSAAI LDAP with:
* SRAM: 70000000 - 72999999
* IDVault: 50100000 - 55999999
* LSAAI: 30000000 - 32999999
and you query all 3 IdPs for user `someuser`
and both LSAAI and IDVault return an entry with UID 5500000,
then the entry from LSAAI is dropped/ignored, because the ID is from the wrong range.